scripts/ebeam_scan_to_multiprobe.py

```diff
@@ -171,20 +140,19 @@ def main(args):
 
     ccd = model.getComponent(role=options.role)
     try:
-        sem_rotation = get_sem_rotation(ccd, auto=options.auto,
-                                        channel=options.channel)
-        if options.auto:
-            scanner = model.getComponent(role="ebeam-control")
+        sem_rotation, scale = get_sem_rotation(ccd)
+        if options.scanner:
+            scanner = model.getComponent(role=options.scanner)
             val = scanner.rotation.value
             scanner.rotation.value = val + sem_rotation
-            print("Added {:.2f}° to the scan rotation using SEM PC."
-                  "Rotation is now set to: {:.2f}°".format(
-                math.degrees(sem_rotation),
-                math.degrees(scanner.rotation.value)))
+            print("Added {:.2f}° to the scan rotation using SEM PC. Rotation is now set to: {:.2f}°".format(
+                    math.degrees(sem_rotation),
+                    math.degrees(scanner.rotation.value)))
         else:
-            print("Add {:.2f}° to the scan rotation using SEM PC. If negative"
-                  "subtract this value from the scan rotation.".format(
-                math.degrees(sem_rotation)))
+            print("Add {:.2f}° to the scan rotation using SEM PC.".format(
+                    math.degrees(sem_rotation)))
+        print("Gain of the e-beam deflectors is {} pixels per volt in x and {} pixels per volt in y".format(
```

Comment:
Maybe we should add somewhere what the definition of gain is. I find it still confusing, as I know gain as an amplification value.